ci-operator/step-registry/ipi/install/powervs/install/ipi-install-powervs-install-commands.sh

```diff
@@ -461,6 +463,12 @@ EOF
       ibmcloud pi network-delete ${UUID}
     done
   ) < <(ibmcloud pi networks --json | jq -r '.networks[] | select(.name|test("rdr-multiarch-'${POWERVS_ZONE}'")) | .networkID')
+
+  if ! ${DESTROY_SUCCEEDED}
+  then
+    echo "Failed to destroy cluster failed after three attempts. Above debug output points to a PowerVS problem."
```

Comment:
Should be fine with the following line 469 replacement:
echo "Failed to destroy cluster after three attempts."